File: Python/repetitivas/sequencia_impares.py
# Exercício: Sequência de números ímpares
# Objetivo: exibir os números ímpares menores que o valor informado.

x = int(input("Digite o valor de X: "))

for i in range(1, x):
    if i % 2 != 0:
        print(f"{i}")

File: Java/Estruturas Repetitivas/sequencia_impares.java
// Exercício: Sequência de ímpares
// Objetivo: identificar e exibir todos os números ímpares de 1 até o valor informado.

import java.util.Scanner;

public class sequencia_impares {
    public static void main(String[] args) {
        
        Scanner sc = new Scanner(System.in);

        int x, i;

        System.out.print("Digite os valores de X: ");
        x = sc.nextInt();

        for (i = 1; i <= x; i++) {
            if (i % 2 != 0) {
                System.out.println(i);
            }
        }

        sc.close();
    }
}